src/output-json.c

```diff
@@ -705,6 +705,9 @@ void CreateJSONFlowId(json_t *js, const Flow *f)
     if (f->parent_id) {
         json_object_set_new(js, "parent_id", json_integer(f->parent_id));
     }
+    if (f->tenant_id > 0) {
```

Comment:
nit: Suggest `if (f->tenant)`

Reply:
All other if-statements doing anything with `tenant_id` are using `!= 0`, `== 0` and `> 0` explicitly. Perhaps `!= 0` would be more idiomatic? I originally based this change on the code from `src/out-json-alert.c` which is one of `> 0` users.